package-lock.json

```diff
@@ -0,0 +1,3509 @@
+{
```

Comment:
This file should be added in a separate PR, IMO

Reply:
Removed from commit but still exists because of the base branch.